src/borg/helpers/parseformat.py

```diff
@@ -493,8 +493,9 @@ def with_timestamp(self, timestamp):
         })
 
     def omit_archive(self):
-        #self.orig = self.orig.split("::")[0]
-        return self
+        loc = Location(self.orig)
+        loc.orig = loc.orig.split("::")[0]
+        return loc
```

Comment:
can you please add a unit test that checks whether this works as expected?

also check the value of `loc.archive` within that test.

Reply:
I am trying to figure out where to write the new test. Does testsuite/helpers.py sound good?